### DISCUSSION TITLE
cannot write security rules in firestore

### DISCUSSION THREAD

**Discussion by thanhtutzaw** [gladly-team/next-firebase-auth#586] [Q&A]
### Without Rules ✔️ ( but got warning from firestore)
![Screenshot (99)](https://user-images.githubusercontent.com/71011043/202675581-0e3335b0-6018-40af-90c3-9b3a39064599.png)
![Screenshot (100)](https://user-images.githubusercontent.com/71011043/202675570-30ddbdb7-8ddc-4841-89f5-faaa4c5bc414.png)

-----------------------------------------------------------------------------------------------------------------------------------------------

### With Rules ✖️ ( Stopped working )
![Screenshot (97)](https://user-images.githubusercontent.com/71011043/202675431-25620ce2-2ebf-49ef-8379-84f667fffcc7.png)
![Screenshot (101)](https://user-images.githubusercontent.com/71011043/202676260-e7f29a6f-c3df-4a31-b15f-a60c08011e06.png)

**Comment by thanhtutzaw** [comment]
My security rules works in not used next-firebase-auth web app .

**Reply by Roee-Tsur** [reply]
Any progress?
I'm facing the same issue.
When I make firestore requests in getServerSideProps the request that is sent is unauthenticated.
